timescaledb/tutorials/ingest-real-time-websocket-data.md

```diff
@@ -0,0 +1,346 @@
+# Ingest real-time financial websocket data
+This tutorial shows you how to ingest real-time time-series data into
+TimescaleDB using a websocket connection. The tutorial sets up a data pipeline to
+ingest real-time data from our data partner, [Twelve Data][twelve-data].
```

Comment:
Logo mark is often different to the written mark (like Red Hat, which is always written as two words, even though there's no space between them in the logo). If you have an authority to ask, then we'll go with what they say, but otherwise I'd go with two words, just because that's what I see on their site.

Reply:
Just got the answer from them, it's "Twelve Data"